src/app/menu/menu.component.scss

```diff
@@ -93,4 +92,63 @@
 {
     padding-left:50px;
     transition: padding-left 0.5s;
+}
+
+@media(max-width: 700px){
+    #menuDesktop{
+        display: none;
+    }
+    #menuMobile{
+        display: block;
+    }
+}
+
+@media(min-width: 756px){
+    #menuDesktop{
+        display: block;
+    }
+    #menuMobile{
+        display: none;
+    }
+}
+
+#menuMobile{
+    position: absolute;
+    bottom: 0;
+    
+
+    #bottomNav{
+        background:rgba(49, 70, 91, 0.7);        
+
+        .col-xs-4{
+            border-left: 1px solid rgba(64, 89, 114, 0.7);;
+            padding-top: 8px;
+            padding-bottom: 8px;
+        }
+
+        a{
+             color:#fff;
+             
+
+            &.active{                
+                color:#f3a117;
+
+
+                // PRUEBA DE CIRCULO SIN SENTIDO    
```

Comment:
Remove that kind of comments before commit

Reply:
DONE